aspnet/fundamentals/configuration/sample/src/UsingOptions/Controllers/HomeController.cs

```diff
@@ -16,9 +16,6 @@ public HomeController(IOptions<MyOptions> optionsAccessor)
         MyOptions Options { get; }
 
         // GET: /<controller>/
-        public IActionResult Index()
-        {
-            return View(Options);
-        }
+        public IActionResult Index() => View(Options);
```

Comment:
Considering storing the `IOptions<T>` in the field or property and then calling `.Value` on first usage. That is the pattern we want to show to customers. I.e. think of it as similar to `Lazy<T>`.